src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -347,9 +347,15 @@ const ConfigureEmbedding = (props) => {
 
     if (selectedPlot === 'sample'
       && !cellSets.loading
-      && filterCells(cellSets, selectedConfig.selectedCellSet).length === 0) {
+      && cellSets.hierarchy.find((rootNode) => rootNode.key === 'sample')?.children?.length === 1
```

Comment:
This is an unrelated fix that I did while working on the original problem.

Some plots: frequency and embedding in "Data Integration" and samples in "Configure Embedding" are supposed to be disabled when the experiment has only one sample, however the code used to check for this in Configure Embedding was not working correctly and was nonexistent in "Data Integration". 

I consulted with oliver if this should be like that and then fixed it.

Reply:
Can you refactor this check and rename it to make it easier to read? e.g. `hasOnlyOneSample`